api/v1alpha1/imagepolicy_types.go

```diff
@@ -53,6 +56,18 @@ type SemVerPolicy struct {
 	Range string `json:"range"`
 }
 
+// AlphabeticalPolicy specifices a alphabetical ordering policy.
+type AlphabeticalPolicy struct {
+	// Prefix enables filtering the list of tags by given prefix.
+	// +optional
+	Prefix string `json:"prefix,omitempty"`
+	// Reversed enables using reversed alphabetical order.
+	// If true the order would be: C B A (A would be chosen)
+	// If false the order would be: A B C (C would be chosen)
+	// +optional
+	Reversed *bool `json:"reverse,omitempty"`
```

Comment:
An alternative, more familiar to some people, is using `order: ASC|DESC` (as in "ascending" or "descending"). The intuitive default would be ascending, which is the same as `reverse: false`, the default here and the most useful for e.g., calendar-dated tags.

Reply:
My thinking was based more on the meaning of the adjective, which implies the default ascending order according to the alphabet, so *reversed* came to mind as the right modifier word. I would be inclined to change it if you think *order* makes more sense.